src/test/java/com/fullstackweb_app/fullstackweb_app/HelloWorldControllerTest.java

```diff
@@ -0,0 +1,16 @@
+package com.fullstackweb_app.fullstackweb_app;
+
+import org.junit.Test;
+
+import static org.junit.Assert.assertEquals;
+
+public class HelloWorldControllerTest {
+    @Test
+    public void testCapitalizeString(){
+        assertEquals("This should be capitalized", "THIS SHOULD BE CAPITALIZED", HelloWorldController.capitalizeString("this should be capitalized"));
+
+

```

Comment:
Not sure why there are so many extra newlines here, but you need more than one assertion in your test.